torch/_functorch/compilers.py

```diff
@@ -183,6 +183,10 @@ def debug_nop(fx_g: fx.GraphModule, _) -> Callable:
 @make_boxed_compiler
 def simple_ts_compile(fx_g, _):
     strip_overloads(fx_g)
+
+    # realize the lazy recompilication to make jit.script happy.
+    fx_g.real_recompile()
```

Comment:
This looks suspicious. Are you breaking BC here?

Reply:
I change the behavior that `GraphModule.recompile` will not do the recompilation right away. The recompilation will only be triggered later when we call `__call__`/`forward` or access properties like `code`.

This indeed cause some trouble for torchscript here. Without the ***real*** recompilation, the torchscript will see our ***lazy*** version of the `forward` method and fail. So I force feeding the ***recompiled*** version of `forward` method to torchscript. I think there can be a more transparent way to do this. But I'm not quite familiar with torchscript and not sure where is a good place that I can add code like:

```
if isinstance(mod, fx.GraphModule):
  mod.real_recompile()
```